de-lib/src/main/java/org/iplantc/de/fileViewers/client/callbacks/ShareAnonymousCallback.java

```diff
@@ -60,32 +60,33 @@ public void onFailure(Integer statusCode, Throwable caught) {
     }
 
     @Override
-    public void onSuccess(String result) {
+    public void onSuccess(List<String> urls) {
         if (container != null) {
             container.unmask();
         }
-        JSONObject obj = jsonUtil.getObject(result);
-        JSONObject paths = jsonUtil.getObject(obj, "paths");
-        showShareLink(jsonUtil.getString(paths, file.getPath()));
+
+        showShareLink(urls.stream()
+                          .filter(path -> !diskResourceUtil.isGenomeIndexFile(path))
+                          .collect(Collectors.toList()));
     }
 
 
-    private void showShareLink(String linkId) {
+    private void showShareLink(List<String> linkIds) {
         // Open dialog window with text selected.
         IPlantDialog dlg = new IPlantDialog(true);
         dlg.setHeading(appearance.sendToEnsemblMenuItem());
         attachHelp(dlg);
 
         dlg.setHideOnButtonClick(true);
         dlg.setResizable(false);
-        dlg.setSize("535", "175");
+        dlg.setSize("640", "320");
```

Comment:
We should probably use an appearance class for these size values.  But if you're trying to get this in for Monday's drop, it's not a huge deal.